source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml

```
@@ -0,0 +1,126 @@
+title: Configure Your KMIP KMS
+ref: configure-kmip-kms
+content: |
+
+  To connect a MongoDB driver client to your KMIP KMS, you must configure your KMS
+  such that it has the following behaviors:
+
+  - Accepts your client's TLS certificate
+  - Contains a symmetric master key
+
+  Consult the documentation for your KMIP KMS for information on how
+  to perform these tasks for your specific KMS server.
+
+---
+title: Specify your Certificates
+ref: specify-certificates
+content: |
+
+  Your client must connect to your KMIP KMS through TLS and present
+  a client certificate accepted by your KMS server.
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        Specify the following Java system properties to configure your client's
+        TLS connection: 
+
+        .. code-block:: Shell
+        
+           -Djavax.net.ssl.enabled=true
+           -Djavax.net.ssl.keyStoreType=pkcs12
+           -Djavax.net.ssl.keyStore=<path to pkcs12 KeyStore containing your client certificate>
+           -Djavax.net.ssl.keyStorePassword=<KeyStore password>
+           -Djavax.net.ssl.trustStoreType=jks
+           -Djavax.net.ssl.trustStore=<path to TrustStore containing KMIP KMS CA file>
+           -Djavax.net.ssl.trustStorePassword=<TrustStore password>
+      
+        .. note:: Configure Client With SSLContext
+
+           If you would rather configure your KMIP client using an SSL context, use the 
+           `kmsProviderSslContextMap <{+java-api+}/apidocs/mongodb-driver-core/com/mongodb/ClientEncryptionSettings.Builder.html#kmsProviderSslContextMap(java.util.Map)>`__
+           method.
+
+---
+title: Create a New Data Encryption Key
+ref: create-a-new-data-key
+content: |
+  To encrypt your data, you need a data encryption key generated from your
+  KMS-hosted **master key**. The following diagram shows the requests you need
+  to make from the client application to create and store a new **data
+  encryption key**:
+
+  .. image:: /figures/CSFLE_Data_Key_KMS.png
+     :alt: Diagram that describes creating a data encryption key when using a KMS provider
+
+  1. First, specify the following information to access the master key:
+
+     .. list-table::
+        :header-rows: 1
+        :stub-columns: 1
+        :widths: 30 15 45
+
+        * - Field
+          - Required
+          - Description
+
+        * - keyId
+          - Yes
+          - The ``keyId`` field of your KMIP object.
+
+        * - endpoint
+          - Yes
+          - The URI of your KMIP server.
+
+  2. Once you have the required information, update and run the following code
+     to generate the new data encryption key:
+     
+  .. include:: /includes/substitute-placeholders.rst
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        .. code-block:: Java
```

Syntax highlighting is not showing on the staged documentation. This is my guess based on other examples:

```suggestion
        .. code-block:: java
```

---

🤦